vault_password_file

```diff
@@ -0,0 +1,11 @@
+#/bin/bash
+#
+# Change VAULT to the location of the vault password file 
+# Vault password is not required for dev VMs.
```

Comment:
Does `~/.hisparc_vault` really not work on windows?
Otherwise I would suggest removing this comment, that should just be the location of your vault password, otherwise someone might change it for their local situation and accidentally commit that.

Reply:
I does work on windows! +1

(Because the `ansible_local` provisioner actually runs this on CentOS on the guest box)